vm-virtio/src/fs.rs

```diff
@@ -0,0 +1,409 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
```

Comment:
Is this correct for this whole file?

Reply:
Not sure. Well I mean no other project has this file or this code. I started it from scratch but of course it is inspired from the same model that we followed from Firecracker/Crosvm codebase. Do you think I should include their copyright?

Reply:
This is fine then.